pubspec.yaml

```diff
@@ -15,7 +15,7 @@ screenshots:
     path: Messaging.png
 
 environment:
-  sdk: ^3.5.0
+  sdk: ^3.0.0
   flutter: ">=3.7.0"
```

Comment:
If minimum Dart sdk version is 3.0.0, than the minimum Flutter sdk version should be ^3.10.0.

According Flutter release information: https://flutter-ko.dev/development/tools/sdk/releases.
Minimum Flutter sdk version which supports Dart sdk version 3.0.0 is 3.10.0.

Reply:
Updated